API.md

```diff
@@ -8,27 +8,16 @@ This version requires **joi** v17 or newer.
 const Joi = require('joi')
     .extend(require('@joi/date'));
 
-const schema = Joi.date().format('YYYY-MM-DD').utc();
+const schema = Joi.date().format('YYYY-MM-DD');
```

Comment:
Bad example, you are parsing 'local week-numbering year' and 'day of a year', which doesn't make sense.